GoalieStatsTracker/Views/RecordStatsView.swift

```diff
@@ -124,142 +124,144 @@ struct RecordStatsView: View {
     }
     
     var body: some View {
-        VStack {
+        ScrollView(.vertical) {
```

Comment:
There's a lot of change below, but all I have done is add a `ScrollView(.vertical) {}`.